plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/BlueOceanHook.java

```diff
@@ -47,4 +42,15 @@ protected String getParentUrl() {
     protected String getParentProjectName() {
         return "blueocean-parent";
     }
+
+    @Override
+    public boolean check(Map<String, Object> info) throws Exception {
+        return isBOPlugin(info);
+    }
+
+    public static boolean isBOPlugin(Map<String, Object> moreInfo) {
+        PomData data = (PomData) moreInfo.get("pomData");
+        return data.parent.artifactId.equalsIgnoreCase("blueocean-parent");
```

Comment:
any possibility of data or data.parent being null?

Reply:
Not AFAIK in the case of BO and DP